chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/listing/pending/PendingCertifiedProductDTO.java

```diff
@@ -111,6 +111,7 @@
     private Boolean accessibilityCertified;
     private TransparencyAttestationDTO transparencyAttestation;
     private String transparencyAttestationUrl;
+    private String svapNoticeUrl;
```

Comment:
I think this value needs to get set in one of the constructors... probably the one that is using the details object because that's what fills in the DTO's fields with data from a request sent to the API.